recipe/meta.yaml

```diff
@@ -10,50 +10,129 @@ source:
   sha256: 0715d4a0393437aa2a48c144756360f9b5292635a05fd15c38cbbaf04ef5acb9
 
 build:
-  entry_points:
-    - connexion = connexion.cli:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
-requirements:
-  host:
-    - python {{ python_min }}
-    - poetry-core >=1.2.0
-    - pip
+outputs:
+  - name: connexion-base
+    build:
+      entry_points:
+        - connexion = connexion.cli:main
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    requirements:
+      host:
+        - python {{ python_min }}
+        - poetry-core >=1.2.0
+        - pip
+      run:
+        - typing_extensions >=4.6.1
+        - python >={{ python_min }},<4.0.0
+        - asgiref >=3.4
+        - httpx >=0.23
+        - inflection >=0.3.1
+        - jsonschema >=4.17.3
+        - jinja2 >=3.0.0
+        - python-multipart >=0.0.15
+        - pyyaml >=5.1
+        - requests >=2.27
+        - starlette >=0.35
+        - typing-extensions >=4.6.1
+        - werkzeug >=2.2.1
+      run_constrained:
+        - a2wsgi >=1.7
+        - flask >=2.2
+        - swagger-ui-bundle >=1.1.0
+        - uvicorn >=0.17.6
+        - jsf >=0.10.0
 
-  run:
-    - typing_extensions >=4.6.1
-    - python >={{ python_min }},<4.0.0
-    - asgiref >=3.4
-    - httpx >=0.23
-    - inflection >=0.3.1
-    - jsonschema >=4.17.3
-    - jinja2 >=3.0.0
-    - python-multipart >=0.0.15
-    - pyyaml >=5.1
-    - requests >=2.27
-    - starlette >=0.35
-    - typing-extensions >=4.6.1
-    - werkzeug >=2.2.1
-  run_constrained:
-    - a2wsgi >=1.7
-    - flask >=2.2
-    - swagger-ui-bundle >=1.1.0
-    - uvicorn >=0.17.6
-    - jsf >=0.10.0
+    test:
+      imports:
+        - connexion
+        - connexion.apps
+        - connexion.decorators
+      commands:
+        - connexion --version
+        - pip check
+      requires:
+        - pip
+        - python {{ python_min }}
+
+  - name: connexion
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+      run:
+        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
```

Comment:
Here and elsewhere:
```suggestion
      run:
        - python >={{ python_min }}
        - {{ pin_subpackage('connexion-base', max_pin="x.x.x") }}
```

Reply:
Because `python` has an appropriate `run-exports` on the `host:` block this should be unnecessary, right?
https://prefix.dev/channels/conda-forge/packages/python

Reply:
python doesn't have run-exports.  The host value doesn't affect the run value.  In this case, `connexion-base` will provide the right constraint on python but the linter will complain anyway so it's just best to put the python constraint explicitly in each output.

Reply:
![image](https://github.com/user-attachments/assets/c4ab5a0c-d5c1-4043-8d68-9d3666d20eaa)

Am I mis-reading this?

Reply:
It just means that `python` is required at run, bu there is no constraint on the version if you are `noarch`.

Reply:
But, yes, I see what you mean, there is a run-exports.